container/ethereum.go

```diff
@@ -386,6 +387,9 @@ func (eth *ethereum) ConsensusMonitor(errCh chan<- error, quit chan struct{}) {
 			log.Printf("Connection lost: %v", err)
 			errCh <- err
 			return
+		case <-time.After(10 * time.Second):
```

Comment:
how about using `Timer` here?
The `Timer` can be initialized by 10s and reset by 3s while receiving blocks.